src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesIfStatement.java

```diff
@@ -17,8 +17,8 @@
 
     void method(String sectionName) {
         if ("Content".equals(sectionName) || "Overview".equals(sectionName)
-                || (!"AbbreviationAsWordInName".equals(sectionName) // violation
-                                                                    // parenthesis
+                || (!"A".equals(sectionName) // violation 'Unnecessary paren.* around expression'
+                                             // parentheses
```

Comment:
Please do me favor, remove content of this line, it was good hint when we had no support of messages of violation 
It should be empty line now

Reply:
Done